modules-32bit.yml

```diff
@@ -55,8 +55,6 @@ modules:
       arch:
         x86_64: *compat_i386_opts
     buildsystem: cmake-ninja
-    config-opts:
-      - -DGSTREAMER=ON
```

Comment:
Why was this removed?

Reply:
Because [GStreamer support has been removed in FAudio 22.02](https://github.com/FNA-XNA/FAudio/releases/tag/22.02).

Reply:
(and therefore, the option doesn't exist anymore)

Reply:
Ugh. I wonder why we need FAudio at all now, then.

Reply:
I see now that I have missed the i386 counterpart.

Reply:
Maybe we don't? It would be great to remove it, and the s-l-g overrides.

Reply:
I'll remove the relevant commit from this PR and make a new one for it.

Reply:
AFAIK it was needed exactly because it provided proprietary codecs support for Proton. If it doesn't provide it anymore, I'm not sure what else it's useful for.

Reply:
it was removed through second PR so I'm resolving this.